src/ctap2/commands/mod.rs

```diff
@@ -124,7 +125,7 @@ pub(crate) trait PinUvAuthCommand: RequestCtap2 {
     fn set_uv_option(&mut self, uv: Option<bool>);
     fn get_uv_option(&mut self) -> Option<bool>;
     fn get_rp_id(&self) -> Option<&String>;
-    fn set_discouraged_uv_option(&mut self);
+    fn handle_discouraged_uv_option(&mut self, info: &AuthenticatorInfo) -> bool;
```

Comment:
Maybe call this `requires_uv` or something similar that makes it easy to interpret the return value.